src/components/MoneyRequestConfirmationList.js

```diff
@@ -171,7 +171,7 @@ function MoneyRequestConfirmationList(props) {
     const {unit, rate, currency} = props.mileageRate;
     const distance = lodashGet(transaction, 'routes.route0.distance', 0);
     const shouldCalculateDistanceAmount = props.isDistanceRequest && props.iouAmount === 0;
-    const shouldCategoryEditable = !_.isEmpty(props.policyCategories) && !props.isDistanceRequest;
+    const shouldCategoryBeEditable = !_.isEmpty(props.policyCategories);
```

Comment:
We'll need to check if someone is on the beta here as per my comment here:
https://github.com/Expensify/App/issues/24463#issuecomment-1709263744